### DISCUSSION TITLE
Allow multiple styles to be applied by just specifying screen size once?

### DISCUSSION THREAD

**Discussion by daftylooper** [tailwindlabs/tailwindcss#11662] [Ideas]
The way responsive design is made possible in Tailwind is - 
`<div className="sm: flex sm: flex-row sm: items-center"></div>`

It becomes tedious to constantly type out the `sm:`. This affects productivity of a developer.

I hope to see a way to wrap all of these CSS rules withing one screen size like - 
`<div className="sm: (flex flex-row items-center)">/<div>`

This makes it a lot more readable and easy to use!

**Comment by wongjn** [comment]
There have been many suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from a maintainer, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)
